File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        result=[]
        n=len(nums)
        for i in range(0,n):
            for j in range(i+1,n):
                if nums[i]+nums[j]==target:
                    result.append(i)
                    result.append(j)
                    break
        return result

File: 0001-two-sum/0001-two-sum.java
import java.util.*;
class Solution {
    public int[] twoSum(int[] nums, int target) {
       HashMap<Integer,Integer>output=new HashMap<>();
       for (int i=0;i<nums.length;i++){
        int diff=target-nums[i];
        if (output.containsKey(diff)){
            if (i<output.get(diff)){
                return new int[]{i,output.get(diff)};
                }
            else{
                return new int[]{output.get(diff),i};
            }
        }
        output.put(nums[i],i);
       }
    return new int[]{-1,-1};
    }
}